lib/miq_automation_engine/service_models/miq_ae_service_network_router.rb

```diff
@@ -8,8 +8,5 @@ class MiqAeServiceNetworkRouter < MiqAeServiceModelBase
     expose :network_ports,         :association => true
     expose :vms,                   :association => true
     expose :private_networks,      :association => true
-
-    expose :raw_update_network_router
-    expose :raw_delete_network_router
```

Comment:
@gmcculloug any issues with removing these?  I only ask because un-exposing methods might be tricky ...

I wonder if there's a deprecation model in Automate?